killerbee/pcapdump.py

```diff
@@ -203,8 +197,10 @@ def pcap_dump(self, packet, ts_sec=None, ts_usec=None, orig_len=None,
                     raise Exception("Longitude value is out of expected range: %.8f" % lon)
                 if alt > -180000.00005 and alt < 180000.00005:
                     alt_i = int(round((alt + 180000.0) * 1e4))
+                elif math.isnan(alt):
+                    alt_i = 4294967295 
```

Comment:
I didn't write this part. I'm assuming it's a way to catch if alt hasn't been set by the GPS and give it a huge number (2**32-1) to show that it's unknown.